Polyfills/Canvas/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(SOURCES
+    "Include/Babylon/Polyfills/Canvas.h"
+    "Source/Canvas.cpp"
+    "Source/Canvas.h"
+    "Source/Context.cpp"
+    "Source/Context.h"
+    "Source/MeasureText.cpp"
+    "Source/MeasureText.h"
+    "Source/nanovg_babylon.cpp"
+    "Source/nanovg_babylon.h"
+    )
+
+file(GLOB FONT_SOURCES ${BGFX_DIR}/examples/common/font/*.cpp)
+file(GLOB NANOVG_SOURCES ${BGFX_DIR}/examples/common/nanovg/nanovg.cpp)
+set(ATLAS_SOURCES ${BGFX_DIR}/examples/common/cube_atlas.cpp)
```

Comment:
Oy, using code from the examples. Is there a better way to handle this?

Reply:
Another solution is to find each github subrepo et add them to github.

Reply:
@bghgary is using example code here a reason to block this review? or will it suffice for the time being if we file a follow up issue?

Reply:
We don't need to block the review.